### ISSUE TITLE
Kafkacat doesn't compile on Alpine Linux

### ISSUE THREAD

**Issue by jurriaan** [edenhill/kafkacat#70]
Steps to reproduce:

Dockerfile:
```dockerfile
FROM alpine

RUN apk add --update alpine-sdk bash python cmake
RUN curl https://codeload.github.com/edenhill/kafkacat/tar.gz/master | tar xzf - && cd kafkacat-* && bash ./bootstrap.sh
```

```
$ docker build -f Dockerfile .
```

This results in: 
<details>
 <summary>Output</summary>
```
Using -L/usr/local/lib  -lpthread -lrt for rdkafka
Package yajl was not found in the pkg-config search path.
Perhaps you should add the directory containing `yajl.pc'
to the PKG_CONFIG_PATH environment variable
Package 'yajl', required by 'world', not found
grep: tmp-bootstrap/usr/local/lib/pkgconfig/yajl.pc: No such file or directory
Using  for yajl
checking for OS or distribution... ok (Linux)
checking for C compiler from CC env... failed
checking for gcc (by command)... ok
checking executable ld... ok
checking executable nm... ok
checking executable objdump... ok
checking executable strip... ok
checking for pkgconfig (by command)... ok
checking for install (by command)... failed
fatal: Not a git repository (or any of the parent directories): .git
checking for rdkafka (by pkg-config)... failed
checking for rdkafka (by compile)...mktemp: Invalid argument
mv: can't rename '': No such file or directory
 failed (fail)
checking for librdkafka metadata API...mktemp: Invalid argument
mv: can't rename '': No such file or directory
 failed (fail)
checking for librdkafka KafkaConsumer support...mktemp: Invalid argument
mv: can't rename '': No such file or directory
 failed (disable)
checking for yajl (by pkg-config)... failed
checking for yajl (by compile)...mktemp: Invalid argument
mv: can't rename '': No such file or directory
 failed (disable)



###########################################################
###                  Configure failed                   ###
###########################################################
### Accumulated failures:                               ###
###########################################################
 rdkafka () 
      desc: librdkafka is available at http://github.com/edenhill/librdkafka. To quickly download all dependencies and build kafkacat try ./bootstrap.sh
    module: kafkacat
    action: fail
    reason:
compile check failed:
CC: CC
flags: tmp-bootstrap/usr/local/lib/librdkafka.a
gcc  -Itmp-bootstrap/usr/local/include -g -O2 -Wall -Wfloat-equal -Wpointer-arith -Wall -Werror tmp-bootstrap/usr/local/lib/librdkafka.a .c -o .c.o  :
.c: file not recognized: File format not recognized
collect2: error: ld returned 1 exit status
source: #include <librdkafka/rdkafka.h>

 librdkafkaver () librdkafka metadata API
      desc: librdkafka 0.8.4 or later is required for the Metadata API
    module: kafkacat
    action: fail
    reason:
compile check failed:
CC: CC
flags: 
gcc  -Itmp-bootstrap/usr/local/include -g -O2 -Wall -Wfloat-equal -Wpointer-arith -Wall -Werror  .c -o .c.o  :
.c: file not recognized: File format not recognized
collect2: error: ld returned 1 exit status
source: #include <librdkafka/rdkafka.h>
struct rd_kafka_metadata foo;

###########################################################
### Installing the following packages might help:       ###
###########################################################
sudo apt-get install  librdkafka-dev
```
</details>

**Comment by edenhill** [comment]
I updated to a newer version of mklove with some mktemp portability fixes, can you give latest kafkacat master a try?

**Comment by jurriaan** [comment]
Wow, that's quick! It works now, thanks a lot!

**Comment by eterps** [comment]
Thanks!

**Comment by AlbertoCastelo** [comment]
I was able to build Kafkacat in a docker container following similar configuration and OS as above. However, I need SSL support. In this case, it did not build that configuration correctly, which it did when I built it in a Ubuntu system. 

With Alpine shows: `librdkafka 0.11.4 builtin.features=snappy,sasl,regex,lz4,sasl_plain,plugins`

Also I checked the flags in librdkafka's config.h (ENABLE_SSL=1). 

How can I enable SSL communication?
